docs/source/getting_started.rst

```diff
@@ -171,7 +313,28 @@ library module that you import. Some modules ship with stubs that mypy
 can automatically find, or you can install a 3rd party module with
 additional stubs (see :ref:`installed-packages` for details).  You can
 also :ref:`create stubs <stub-files>` easily. We discuss ways of
-silencing complaints about missing stubs in :ref:`existing-code`.
+silencing complaints about missing stubs in :ref:`ignore-missing-imports`.
+
+Configuring mypy
+****************
+
+Mypy supports many command line options that you can use to tweak how
+mypy behaves.  They are documented in :ref:`command-line`.
+
+For example, suppose you want to make sure *all* functions within your
+codebase are using static typing and make mypy report an error if you
+add a dynamically-typed function by mistake. You can make mypy do this
+by running mypy with the ``--disallow-untyped-defs`` flag.
+
+One particularly useful command is the ``--strict`` flag. Running
```

Comment:
I'm not sure if we should recommend `--strict` here. Even mypy doesn't use all the options it enables yet. At least I think that there is too much emphasis on it now. I think it might be okay to mention it here as option for people who want to go the extra mile.